torchbenchmark/models/speech_transformer/config.py

```diff
@@ -0,0 +1,132 @@
+import os
+import json
+import torch
+import kaldi_io
+import dataclasses
+
+from decoder import Decoder
+from encoder import Encoder
+from solver import Solver
+from transformer import Transformer
+from transformer.optimizer import TransformerOptimizer
+from utils import add_results_to_json, process_dict
+from data import build_LFR_features
+from data import AudioDataLoader, AudioDataset
+
+@dataclasses.dataclass
+class SpeechTransformerTrainConfig:
+    # Low Frame Rate
+    LFR_m = 4
+    LFR_n = 3
+    # Network Architecture - Encoder
+    d_input = 80
+    n_layers_enc = 6
+    n_head = 8
+    d_k = 64
+    d_v = 64
+    d_model = 512
+    d_inner = 2048
+    dropout = 0.1
+    pe_maxlen = 5000
+    d_word_vec = 512
+    n_layers_dec = 6
+    tgt_emb_prj_weight_sharing = 1
+    label_smoothing = 0.1
+    # minibatch
+    shuffle = 1
+    batch_size = 16
+    batch_frames = 15000
+    maxlen_in = 800
+    maxlen_out = 150
+    num_workers = 4
+    # optimizer
+    k = 0.2
+    warmup_steps = 1
+    # solver configs
+    epochs = 1
+    save_folder = "output_data"
+    checkpoint = False
+    continue_from = False
+    model_path = 'final.pth.tar'
+    print_freq = 10
+    visdom = 0
+    visdom_lr = 0
+    visdom_epoch = 0
+    visdom_id = 0
+    # input files
+    train_json = "input_data/train/data.json"
```

Comment:
Are these paths relative to torchbench? I'm not a big fan of hardcoded paths in general, but I also don't see a better solution :)

Perhaps at least comment so they are clearly called out.

Reply:
These paths are relative to the directory of `__file__`, I will add comments.